ansible/roles/os_zabbix/vars/template_openshift_master.yml

```diff
@@ -2,6 +2,18 @@
 g_template_openshift_master:
   name: Template Openshift Master
   zitems:
+  - key: origin-master.version
+    description: Show origin-master rpm version
+    value_type: string
+    applications:
+    - Openshift Cluster
+
+  - key: atomic-openshift-master.version
```

Comment:
I am sending rpm versions. So the pattern is the `<name of the rpm>.version`.
It also needs to be verified against expected values you push to manageIQ using ansible playbooks.
If hawkular that allows searching by tags, and manageIQ are the future, is it really worth complicating the keys to match a pattern in zabbix, we are planning to replace anyway?